src/main/java/sg/ncl/MainController.java

```diff
@@ -1525,6 +1565,73 @@ public String editTeamProfile(
         return "redirect:/team_profile/" + teamId;
     }
 
+    @RequestMapping(value = "/team_quota/{teamId}", method = RequestMethod.POST)
+    public String editTeamQuota(
+            @PathVariable String teamId,
+            @ModelAttribute("teamQuota") TeamQuota editTeamQuota,
+            final RedirectAttributes redirectAttributes,
+            HttpSession session) throws IOException {
+
+        final String QUOTA = "#quota";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)